v2/index.html

```diff
@@ -965,27 +965,27 @@ <h4>Accidentals</h4>
             <h4>Rests</h4>
             <div class="issue" data-number="29"></div>
             <div class="issue" data-number="41"></div>
-            <p>Rests for single notes are indicated by <code>-</code> (a minus sign).</p>
-            <table class="simple">
-                <thead>
-                <tr>
-                    <th>Code</th>
-                    <th>Name</th>
-                </tr>
-                </thead>
-                <tbody>
-                <tr>
-                    <td><code>-</code></td>
-                    <td>single-note rest (preceded by rhythmic value like note names)</td>
-                </tr>
-                <tr>
-                    <td><code>=</code></td>
-                    <td>measure rest (followed by number of measures and a bar line)</td>
-                </tr>
-
-                </tbody>
-            </table>
-
+            <p>
+                Rests for single notes MUST be indicated by a hyphen/minus character <code>-</code>. This character
+                MUST be preceded by a <a>duration</a> value giving the musical duration of the rest.
+            </p>
+            <p>
+                Measure rests MUST be indicated by an equal sign character <code>=</code>. This character MUST be
+                followed by a non-negative integer indicating the number of measures for which this rest applies,
+                unless the measure rest only applies to a single measure. In this case, the number MAY be omitted.
+            </p>
+            <p>
+                Measure rests MUST be followed by a <a>bar line</a> character.
+            </p>
+            <p>
+                Measure rests MUST NOT be used with Mensural notation, due to the general absence of measures in
+                this notation.
```

Comment:
Fine for now, but might need to revise this depending on some clarifications for the mensural notation.

Reply:
Which clarifications?

Reply:
What do we want to do with mensural notation in the RISM dataset. This would make some incipits in RISM invalid, so this is a problem.

Reply:
It would make them invalid to version 2 of the spec, but not version 1. We can either keep those at V1, or upgrade them to V2 and add a bar line.